### ISSUE TITLE
Global SEO Title Issue for a Multi-Site Setup

### ISSUE THREAD

**Issue by richhayler** [nystudio107/craft-seomatic#572]
It appears configuring the Global SEO Title Source to use a global custom field on a multi-site setup is referencing the value of the primary site instead of the selected site.

Importantly the global custom field in question has the translation method set to "Translate for each site" so the field value should be rendered uniquely for each site. I can confirm the values of this field are set uniquely for the global set.

**Comment by khalwat** [comment]
Can you show me some code in terms of what you're actually doing?

SEOmatic just tells Twig to parse the field you've passed in, so it'd be a little strange if this wasn't evaluating properly.

Does the global evaluate properly if you check it in your templates? Have you tried clearing caches?

**Comment by richhayler** [comment]
Thanks for looking into this one for me....

I've attached a few screenshots to illustrate the issue I'm running into..

1. Custom field named "Brand" set to "Translate for each site" - I have added this as a global
<img width="362" alt="1" src="https://user-images.githubusercontent.com/2505643/75982502-e997c100-5f3a-11ea-8e59-992691f3150c.png">

2. Value for primary site is "Hayler Media"
<img width="406" alt="Screenshot 2020-03-05 23 36 58" src="https://user-images.githubusercontent.com/2505643/75982550-016f4500-5f3b-11ea-9332-8308c36a2f7e.png">

3. Value for another site (professional) is "Richard Hayler"
<img width="418" alt="Screenshot 2020-03-05 23 36 42" src="https://user-images.githubusercontent.com/2505643/75982590-16e46f00-5f3b-11ea-89b9-e53d6bc9c762.png">

4. Setting Global SEO Title for site "Professional" and selecting "Brand" as the field to reference.... the value "Hayler Media" is displayed instead of "Richard Hayler".
<img width="521" alt="Screenshot 2020-03-05 23 39 50" src="https://user-images.githubusercontent.com/2505643/75982674-3da2a580-5f3b-11ea-8726-7c316f4ac62c.png">

5. If I output `{{ global.brand }}` in a template on the "Professional" site the correct value "Richard Hayler" is displayed.
